tests/Neo.UnitTests/SmartContract/Manifest/UT_ContractGroup.cs

```diff
@@ -45,7 +45,7 @@ public void TestIsValid()
         ContractGroup contractGroup = new()
         {
             PubKey = keyPair.PublicKey,
-            Signature = new byte[20]
+            Signature = new byte[64]
```

Comment:
general fix on this UT, right? not related to this PR @shargon

Reply:
If it's 20, we won't be able to test the code we want; it will judge from the beginning that the parameter does not meet the format requirements.